core/templates/dev/head/app.js

```diff
@@ -107,6 +107,28 @@ oppia.config(['$provide', function($provide) {
   }]);
 }]);
 
+oppia.config(['toastrConfig', function(toastrConfig) {
```

Comment:
This config is common for all the toasts, so this might be an issue if we use the same kind of toasts for `infoMessages` and `warnings`(unless we plan to display both of them in the same place, then top-right would be an apt place to do so)